server/middlelayer-interaction.es6

```diff
@@ -1,36 +1,51 @@
 'use strict';
 
 let request = require('request');
-
 let util = require('./util.es6');
 let vcapServices = require('./vcap-services.es6');
 
-module.exports = (application, successCallback, failureCallback) => {
-  let authOptions = {
+let middlelayerFacade = {};
```

Comment:
Also it seems like this could be moved into a `Class`, since this is es6. Although, that might not be consistent.